vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -672,6 +675,86 @@ public void afterTextChanged(Editable s) {
 
         mTextInputLayouts = ViewUtilKt.findAllTextInputLayout(mFormContainer);
         ViewUtilKt.autoResetTextInputLayoutErrors(mTextInputLayouts);
+
+
+        mLoginEmailTextView.setOnFocusChangeListener((v, hasFocus) -> {
+            if (hasFocus) return;
```

Comment:
I'm wondering if this is called when we switch to create account mode (?)

Reply:
Tested, it's not called (it's another text view for register)

Reply:
ok